docs/user_guide/architecture.rst

```diff
@@ -0,0 +1,128 @@
+.. _architecture:
+
+Testbenches Architecture
+===============================================================================
+
+File structure of a project
+-------------------------------------------------------------------------------
+
+.. tip::
+
+   In ``/testbenches/tree/main/base/`` you can find a test bench base design.
```

Comment:
```suggestion
   In ``base/`` you can find a test bench base design.
```
Still, would need to update to the new proposed structure, perhaps `projects/base` ?

Reply:
Leaving this question open.